ci/deploy-gh-pages.sh

```diff
@@ -28,12 +24,11 @@ else
 fi
 
 echob "Build doxygen documentation."
-if ! ( cd $script_path && doxygen "Doxyfile" ); then
-    echo "- Doxygen generation failed."
-    exit 1
-fi
+test cd $doc_path
+test doxygen ${doc_path}/Doxyfile
```

Comment:
This gave me an error here as it tried `doxygen doc/Doxyfile` when it was already in `doc` folder.

Reply:
the path for the docs in the rest of this file needs attention too.

Reply:
done